supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -220,7 +221,7 @@ Alternative names: *CA server*
 ==== image:images/yes.png[yes] IdM server and replicas (noun)
 *Description*: In Red Hat Enterprise Linux, to install the first server in an IdM deployment, you must use the `ipa-server-install` command.
 
-Administrators can then use the `ipa-replica-install` command to install *replicas* in addition to the first server that was installed. By default, installing a replica creates a replication agreement with the IdM server from which it was created, enabling receiving and sending updates to the rest of IdM.
+Administrators can then use the `ipa-replica-install` command to install _replicas_ in addition to the first _server_ that was installed. By default, installing a replica creates a replication agreement with the IdM server from which it was created, enabling receiving and sending updates to the rest of IdM.
```

Comment:
This one is surely beyond scope so maybe it could be addressed at a later date.

But this whole entry doesn't seem to define one term like the rest of the glossary does. The part of the entry that contains the two terms (now italicized), doesn't define them, which is when we're supposed to use italics -in a definition. Instead the entry discusses the commands needed to install these servers.
Only the final line of the entry is a definition  "Both are fully functional read/write IdM servers."

Again, not a PR-blocking issue :]

Reply:
Thanks @ndeevy. Maybe you could write this up as a separate issue?

Reply:
Ack. A future issue to review these types of things makes sense.